data/mp/multiplay/skirmish/semperfi_includes/build.js

```diff
@@ -194,21 +194,21 @@ function scanAndDefendPosition(structure, droid)
 function lookForOil()
 {
 	const UNSAFE_AREA_RANGE = 7;
-	var droids = enumGroup(oilBuilders);
-	var oils = enumFeature(-1, OIL_RES_STAT).sort(sortByDistToBase); // grab closer oils first;
-	var bestDroid = null;
-	var bestDist = 99999;
-	var success = false;
+	const droids = enumGroup(oilBuilders);
+	const oils = enumFeature(-1, OIL_RES_STAT).sort(sortByDistToBase); // grab closer oils first;
+	let bestDroid = null;
+	let bestDist = 99999;
+	let success = false;
 	//log("looking for oil... " + oils.length + " available");
 
-	for (var i = 0, oilLen = oils.length; i < oilLen; ++i)
+	for (let i = 0, oilLen = oils.length; i < oilLen; ++i)
 	{
-		for (var j = 0, drLen = droids.length; j < drLen; ++j)
+		for (let j = 0, drLen = droids.length; j < drLen; ++j)
 		{
-			var droid = droids[j];
-			var oil = oils[i];
-			var dist = distBetweenTwoPoints(droid.x, droid.y, oil.x, oil.y);
-			var unsafe = enumRange(oil.x, oil.y, UNSAFE_AREA_RANGE, ENEMIES, false).filter(isUnsafeEnemyObject);
+			const droid = droids[j];
+			const oil = oils[i];
```

Comment:
Something I noticed during testing: `oil` should move a block above for line 226 to work. Otherwise error spam will happen.